cms/djangoapps/contentstore/utils.py

```diff
@@ -430,35 +430,34 @@ def get_visibility_partition_info(xblock):
     Returns: dict
 
     """
-    user_partitions = get_user_partition_info(xblock, schemes=["verification", "enrollment_track", "cohort"])
-    cohort_partitions = []
-    verification_partitions = []
-    enrollment_mode_partitions = []
-    has_selected_content_groups = False
-    has_selected_enrollment_modes = False
-    selected_verified_partition_id = None
-
-    # Pre-process the partitions to make it easier to display the UI
-    for p in user_partitions:
-        if p["scheme"] == "cohort":
-            cohort_partitions.append(p)
-            has_selected_content_groups = any(g["selected"] for g in p["groups"])
-        elif p["scheme"] == "verification":
```

Comment:
Look @efischer19, I found more verification user partition changes I can delete! I'm removing some of the Studio code as part of my current work because I stumbled across it, and some of it I would need to modify if it stays in.